openSUSEway.spec

```diff
@@ -20,59 +20,92 @@
 %define waybar_version %(rpm -q --queryformat "%%{version}" waybar)
 
 Name:           openSUSEway
-Version:        0.15.2
+Version:        0.16.0
 Release:        0
 Summary:        The openSUSEway desktop environment meta package
 License:        MIT
 Group:          Metapackages
 URL:            https://github.com/openSUSE/openSUSEway
 Source0:        https://github.com/openSUSE/openSUSEway/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
-# PATCH-FIX-OPENSUSE add-configuration-for-play-pause-next-prev-multimedi.patch gh#openSUSE/openSUSEway#41 mcepl@suse.com
-# Add multimedia keys configuration
+
 BuildArch:      noarch
 BuildRequires:  aaa_base
 BuildRequires:  pkgconfig(systemd)
+
+# system
+Requires:       wget
+Requires:       glibc-locale
+Requires:       bash-completion
 Requires:       NetworkManager
 Requires:       aaa_base
-Recommends:     adwaita-qt5
-Recommends:     bluez
 Requires:       bzip2
 Requires:       command-not-found
 Requires:       curl
-Recommends:     firefox
-Requires:       gfxboot-branding-openSUSE
 Requires:       git
-Requires:       greetd
 Requires:       grep
-Recommends:     grim
 Requires:       gzip
-Requires:       (gtkgreet or wlgreet)
-Suggests:       imv
 Requires:       jq
 Requires:       less
-Recommends:     libqt5-qtwayland
-Suggests:       mpv
-Requires:       pipewire
-Recommends:     qt5ct
 Requires:       sudo
-Requires:       sway-branding-openSUSE
 Requires:       tar
+
+# basic DE
+Requires:       greetd
+Requires:       (gtkgreet or wlgreet)
+Requires:       pipewire
+Requires:       sway-marker
+Recommends:     grim
+Recommends:     slurp
+Recommends:     firefox
+Recommends:     bluez
 Recommends:     tlp
+Suggests:       mpv
 Suggests:       vifm
 Suggests:       vim
+Suggests:       imv
+
+# basic multi-media
 Requires:       clipman
 Requires:       wl-clipboard
 Requires:       mpris-ctl
-Recommends:     slurp
-Requires:       sway-marker
+
+# branding
 Requires:       waybar-branding-openSUSE
-Requires:       wget
+Requires:       sway-branding-openSUSE
+Requires:       gfxboot-branding-openSUSE
+
+# xdg portals and utils
 Requires:       xdg-desktop-portal
 Requires:       xdg-desktop-portal-wlr
+Requires:       xdg-desktop-portal-gtk
 Requires:       xdg-utils
 
+# Appearance
+Requires:       adwaita-icon-theme
+Requires:       gtk3-metatheme-adwaita
+Requires:       metatheme-adwaita-common
+Recommends:     libqt5-qtwayland
```

Comment:
Not quite sure what those qt packages are doing. But most likely there is a good reason they got added I guess ;)

Reply:
qt5ct is used to set dark theme for QT based apps, and some of those are needed for it. Just better QT experience under wayland. At least was when I added those long time ago :)